test/e2e/settings/disable_catch_all.go

```diff
@@ -105,10 +117,10 @@ var _ = framework.IngressNginxDescribe("[Flag] disable-catch-all", func() {
 			Status(http.StatusNotFound)
 	})
 
-	ginkgo.It("should allow Ingress with both a default backend and rules", func() {
+	ginkgo.It("should allow Ingress with rules", func() {
```

Comment:
thanks for this, i did not realize this test suite had no coverage for happy path

Reply:
np!